src/mbgl/map/tile_worker.cpp

```diff
@@ -64,29 +65,6 @@ void TileWorker::redoPlacement(float angle, float pitch, bool collisionDebug) {
 }
 
 template <typename T>
-struct PropertyEvaluator {
```

Comment:
`PropertyEvaluator` was duplicated in two places and another part of this commit is pulling it out to `mbgl/style/property_evaluator.hpp`.